inventory/sample-ha.aws.example.com.d/inventory/ec2.ini

```diff
@@ -0,0 +1,190 @@
+#{{ ansible_managed | Ansible EC2 external inventory script settings }}
```

Comment:
Why do we have both this .ini and a template for it? Could we eliminate this one and just keep the template?

Reply:
My mistake. As you can see there is a template directory under sample inventory folder as the initial idea was to generate the .ini file using a jinja template, but as this .ini file is being used by the 'ansible-playbook' execution itself there is a 'chicken and egg' problem there, so we need to figure out how to include this (if we finally think it make sense).